src/BaseClient.php

```diff
@@ -68,8 +71,13 @@ private function getHttpClientFromConfig(array $config)
         
         return $client;
     }
-    
-    private function getDescriptionFromConfig(array $config)
+
+    /**
+     * @param array $config
+     * @return Description|mixed
+     * @throws \Exception
+     */
+    private function getDescriptionFromConfig(array $config): Description
```

Comment:
This return type might cause you problems if the `@return Description|mixed` is actually accurate, as PHP will throw a type error if something other than a Description is returned with this return type in place.